css/main.css

```diff
@@ -347,6 +347,90 @@ body {
 
 /* ––––––––– END PARTNERSHIP ––––––––– */
 
+/* ––––––––– JOIN ––––––––– */
+
+.join {
+  background: url('/images/join-bg.png') no-repeat center var(--c-main-body);
+  ;
+  background-size: cover;
+  border-radius: 250px 0px 0px 0px;
```

Comment:
Please remove "px" next to 0. If it's zero – it doesn't matter if it zero pixels or, say, zero cars – it's zero anyway :)